roofit/roofitcore/test/TestStatistics/testPlot.cpp

```diff
@@ -0,0 +1,92 @@
+/*
+ * Project: RooFit
+ * Authors:
+ *   ZW, Zef Wolffs, NIKHEF, zefwolffs@gmail.com
+ *
+ * Copyright (c) 2021, CERN
+ *
+ * Redistribution and use in source and binary forms,
+ * with or without modification, are permitted according to the terms
+ * listed in LICENSE (http://roofit.sourceforge.net/license.txt)
+ */
+
+#include "RooFit.h"
+#include "RooWorkspace.h"
+#include "RooPlot.h"
+#include "RooDataSet.h"
+#include "RooFit/TestStatistics/buildLikelihood.h"
+#include "RooAbsPdf.h"
+#include "RooRealVar.h"
+#include "RooFit/TestStatistics/RooRealL.h"
+#include "RooFit/MultiProcess/Config.h"
+#include "RooMinimizer.h"
+#include <memory>
+
+#include "RooUnitTest.h"
+#include "gtest/gtest.h"
+
+using namespace RooFit ;
+
+class TestRooRealLPlot : public RooUnitTest
+{
+public:
+  TestRooRealLPlot(TFile* refFile, Bool_t writeRef, Int_t verbose, std::string const& batchMode) : RooUnitTest("Plotting and minimization with RooFit::TestStatistics",refFile,writeRef,verbose,batchMode) {} ;
```

Comment:
```suggestion
  TestRooRealLPlot(TFile* refFile, bool writeRef, int verbose, std::string const& batchMode) : RooUnitTest("Plotting and minimization with RooFit::TestStatistics",refFile,writeRef,verbose,batchMode) {} ;
```
You only use `Int_t` for the declaration of persistent data members, i.e. for doing IO across platforms. The feature of `Int_t` is that it's guaranteed to be 32 bit, but here that doesn't matter.